week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -1,17 +1,15 @@
 // Fix Functions
 
-// Aim: to understand the change code inside functions 
+// Aim: to understand the change code inside functions
 //
 // The below functions are syntactically correct but not outputting the right results.
 // Look at the tests and see how you can fix them.
 
-
-
 // 1) mood function does this return `"I am not happy"`
 //    Only make edits inside the function
 
 function mood() {
-  let isHappy = true;
+  let isHappy = false;
 
   if (isHappy) {
     return "I am happy";
```

Comment:
Hedyeh, you need to add: 
} else {
    return "I am not happy";
  }
after line 15